main.py

```diff
@@ -7,20 +7,20 @@
 from ballworld.world import World as BallWorld
 from simplegrid.world import World as GridWorld
 
-WORLDS = {'ball': BallWorld,
-          'grid': GridWorld}
+WORLDS = {'ball': BallWorld, 'grid': GridWorld}
 
 FRAME_RATE = 60
 TITLE = 'Reinforced Artificial Life'
+WORLD_SIZE = 80
+SCALE = 8
 
 
 def main(WorldClass):
-    display = Display(TITLE, 640, 480)
+    display = Display(TITLE, WORLD_SIZE, SCALE)
```

Comment:
I'm not convinced this is the way to go; I'd imagine that you'd want to specify how big the screen is and then adjust the the scale such that the world fills that screen. By picking SCALE you get an enormous window if you switch back to the ball world for example.